core/types/transaction_signing.go

```diff
@@ -41,7 +41,7 @@ func MakeSigner(config *params.ChainConfig, blockNumber *big.Int, blockTime uint
 	var signer Signer
 	switch {
 	case config.IsIsthmus(blockTime):
-		signer = NewIsthmusSigner(config.ChainID)
+		signer = NewIsthmusSigner(config.ChainID, config.IsL2Blob(blockNumber, blockTime))
 	case config.IsPrague(blockNumber, blockTime) && !config.IsOptimism():
 		signer = NewPragueSigner(config.ChainID)
 	case config.IsCancun(blockNumber, blockTime) && (!config.IsOptimism() || config.IsL2Blob(blockNumber, blockTime)):
```

Comment:
How about this part of logic?  Should `IsL2Blob` check be removed here?

Reply:
I mean the Cancun check

Reply:
I think it still needs to be there, we want to enable `cancunSigner` if L2Blob is activated.

Reply:
Yeah, that depends on our usage. We can remove the Cancun check if we'll enable isthmus in genesis.

Reply:
I think we will enable isthmus in genesis

Reply:
Removed the Cancun check.

Reply:
Do we also want to remove the logic in line-78?

Reply:
good point, done